src/remappings.rs

```diff
@@ -67,8 +67,14 @@ impl Remapping {
 
     /// Removes the `base` path from the remapping
     pub fn strip_prefix(&mut self, base: impl AsRef<Path>) -> &mut Self {
+        // converting a string path to Path and back will lose the trailing slash,
+        // so we need to remember if the path had a trailing slash.
+        let has_trailing_slash = self.path.ends_with('/');
```

Comment:
i think on windows this will be `\` instead of `/`

Reply:
Thanks for your review. I realize the problem is not here. Here all the slashes in the path has already been converted to `/` even for windows. The issue is actually when converting `\` to `/`, the trailing `/` is discarded and we should preserve them. I have patched this with a new commit.